app/models/menu.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Menu < ApplicationRecord
+  enum category: {
+    first_course: FIRST_COURSE = 'first_course',
+    main_course: MAIN_COURSE = 'main_course',
+    drinks_course: DRINKS_COURSE = 'drinks_course'
+  }
+
+  enum day: {
```

Comment:
this just looks weird
ruby weekdays is Sunday-to-Saturday based, so I can see why you didn't use it, but find an appropriate gem then.